model.md

```diff
@@ -0,0 +1,4 @@
+The *verification* problem can be solved by cryptographic signing (implemented in IPFS)  
```

Comment:
I think this fits, but I'm a bit leery of "problem" repeated here a bunch without being able to reference the problem space :)

Reply:
This document isn't space-constrained the way the 3 core docs are, right? so we could re=introduce some language about the problems in an initial paragraph.